lxd/storage_pools_config.go

```diff
@@ -169,6 +169,12 @@ func storagePoolValidateConfig(name string, driver string, config map[string]str
 			}
 		}
 
+		if driver != "ceph" {
```

Comment:
@monstermunchkin we shouldnt need this change, as the new pool backend should reject any unknown keys.